testing/unit_tests/cases_loader.py

```diff
@@ -0,0 +1,242 @@
+from pathlib import Path
+
+import torch
+
+from testing.unit_tests.t_utils import __data_testing_dir__, path_repo_root
+from ivadomed.keywords import LoaderParamsKW, ModelParamsKW, TransformationKW
+
+
+def get_microscopy_loader_parameters():
+    default_loader_parameters = {
+        LoaderParamsKW.PATH_DATA: [str(Path(__data_testing_dir__, "microscopy_png"))],
+        LoaderParamsKW.BIDS_CONFIG: f"{path_repo_root}/ivadomed/config/config_bids.json",
+        LoaderParamsKW.TARGET_SUFFIX: [["_seg-myelin-manual", "_seg-axon-manual"]],
+        LoaderParamsKW.EXTENSIONS: [".png"],
+        LoaderParamsKW.ROI_PARAMS: {"suffix": None, "slice_filter_roi": None},
+        LoaderParamsKW.CONTRAST_PARAMS: {"contrast_lst": []}
+    }
+
+    return default_loader_parameters
```

Comment:
I would recommend to also get the default JSON from the `config.json` root file within the current repo., and OVERWRITE its `loader_params` key with it. 

You will see below comments where when this was not done, you kind of have to redefine the model and transformation keys multiple times.